.github/workflows/prettify-code.yml

```diff
@@ -0,0 +1,22 @@
+name: "Run prettier"
+on: # rebuild any PRs and main branch changes
+  pull_request:
+    branches:
+      - main
+      - "releases/*"
+  push:
+    branches:
+      - main
+      - "releases/*"
```

Comment:
remove releases/* from these things. we don't want to enforce prettier on the releases (when we build it uses another library to build that doesn't necessarily follow formatting and doesn't need to)